SimPEG/Directives.py

```diff
@@ -399,14 +505,15 @@ def endIter(self):
                 self.reg.norms = self.norms
                 print("L[p qx qy qz]-norm : " + str(self.reg.norms))
 
+
             if self.ComboObjFun:
                     for reg in self.reg.objfcts:
                         reg.model = self.invProb.model
 
             else:
                 self.reg.model = self.invProb.model
 
-            self.l2model = self.invProb.model.copy()
+            self.reg.l2model = self.invProb.model.copy()
```

Comment:
can we leave the l2model on the inversion directive instead of the regularization?